File: Motocitizen/src/motocitizen/Activity/AboutActivity.java
package motocitizen.Activity;

import android.content.Intent;
import android.graphics.Color;
import android.os.Bundle;
import android.support.v7.app.ActionBar;
import android.support.v7.app.ActionBarActivity;
import android.text.method.LinkMovementMethod;
import android.view.Menu;
import android.view.View;
import android.webkit.WebView;
import android.widget.Button;
import android.widget.TextView;

import motocitizen.MyApp;
import motocitizen.main.R;
import motocitizen.startup.ChangeLog;
import motocitizen.startup.Preferences;

public class AboutActivity extends ActionBarActivity implements View.OnClickListener {

//Посмотреть http://android-developers.blogspot.in/2013/08/actionbarcompat-and-io-2013-app-source.html

    Button bussinesCardButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        MyApp.setCurrentActivity(this);
        setContentView(R.layout.activity_about);

        ((TextView) this.findViewById(R.id.about_code_version)).setText(getString(R.string.code_version_prefix) + ": " + Preferences.getCurrentVersion());

        ActionBar actionBar = getSupportActionBar();
        actionBar.setDisplayHomeAsUpEnabled(true);

        WebView wv = (WebView) findViewById(R.id.change_log);
        wv.setBackgroundColor(Color.rgb(48, 48, 48));
        wv.loadDataWithBaseURL(null, ChangeLog.getLog(this, true), "text/html", "UTF-8", null);

        bussinesCardButton = (Button) findViewById(R.id.bussinesCardButton);
        bussinesCardButton.setOnClickListener(this);

        TextView url_support = (TextView) findViewById(R.id.about_url_support);
        url_support.setMovementMethod(LinkMovementMethod.getInstance());
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        //getMenuInflater().inflate(R.menu.menu_about, menu);
        return true;
    }

    @Override
    public void onClick(View view) {
        int id = view.getId();
        switch (id) {
            case R.id.bussinesCardButton:
                this.startActivity(new Intent(this, BusinessCardActivity.class));
                break;
        }
    }

    @Override
    protected void onPostResume() {
        super.onPostResume();
        MyApp.setCurrentActivity(this);
    }
}

File: Motocitizen/src/motocitizen/ui/activity/AboutActivity.kt
package motocitizen.ui.activity

import android.graphics.Color
import android.os.Bundle
import android.text.method.LinkMovementMethod
import android.text.util.Linkify
import android.view.ViewGroup
import android.webkit.WebView
import android.widget.Button
import android.widget.LinearLayout
import android.widget.ScrollView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import motocitizen.main.R
import motocitizen.ui.Screens
import motocitizen.ui.changelog.ChangeLog
import motocitizen.utils.goTo

class AboutActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
        // Корневой ScrollView (чтобы содержимое скроллилось)
        val scrollView = ScrollView(this).apply {
            setBackgroundColor(Color.rgb(48, 48, 48))
            layoutParams = ViewGroup.LayoutParams(
                ViewGroup.LayoutParams.MATCH_PARENT,
                ViewGroup.LayoutParams.MATCH_PARENT
            )
        }

        // Вертикальный LinearLayout внутри ScrollView
        val rootLayout = LinearLayout(this).apply {
            orientation = LinearLayout.VERTICAL
            layoutParams = ViewGroup.LayoutParams(
                ViewGroup.LayoutParams.MATCH_PARENT,
                ViewGroup.LayoutParams.WRAP_CONTENT
            )
            val padding = (16 * resources.displayMetrics.density).toInt()
            setPadding(padding, padding, padding, padding)
        }

        scrollView.addView(rootLayout)

        // TextView: about_info
        val aboutInfoText = TextView(this).apply {
            setText(R.string.about_info)
            setTextColor(Color.WHITE)
        }
        rootLayout.addView(aboutInfoText)

        // TextView: версия приложения
        val versionName = packageManager.getPackageInfo(packageName, 0).versionName
        val versionText = TextView(this).apply {
            text = getString(R.string.code_version_prefix) + ": $versionName"
            setTextColor(Color.WHITE)
            setPadding(0, (8 * resources.displayMetrics.density).toInt(), 0, 0)
        }
        rootLayout.addView(versionText)

        // TextView: about_url_support (с активными ссылками)
        val supportUrl = TextView(this).apply {
            setText(R.string.about_url_support)
            setTextColor(Color.CYAN)
            movementMethod = LinkMovementMethod.getInstance()
            setPadding(0, (8 * resources.displayMetrics.density).toInt(), 0, 0)
            // Чтобы ссылки были кликабельны, нужно еще autoLink:
            autoLinkMask = Linkify.WEB_URLS
        }
        rootLayout.addView(supportUrl)

        // TextView: about_authors
        val authorsText = TextView(this).apply {
            setText(R.string.about_authors)
            setTextColor(Color.WHITE)
            setPadding(0, (8 * resources.displayMetrics.density).toInt(), 0, 0)
        }
        rootLayout.addView(authorsText)

        // Кнопка business_card
        val businessCardBtn = Button(this).apply {
            setText(R.string.business_card)
            setPadding(0, (16 * resources.displayMetrics.density).toInt(), 0, 0)
            layoutParams = LinearLayout.LayoutParams(
                ViewGroup.LayoutParams.MATCH_PARENT,
                ViewGroup.LayoutParams.WRAP_CONTENT
            )
            setOnClickListener {
                goTo(Screens.BUSINESS_CARD)
            }
        }
        rootLayout.addView(businessCardBtn)

        // WebView для ChangeLog
        val webView = WebView(this).apply {
            layoutParams = LinearLayout.LayoutParams(
                ViewGroup.LayoutParams.MATCH_PARENT,
                ViewGroup.LayoutParams.WRAP_CONTENT
            ).apply {
                topMargin = (16 * resources.displayMetrics.density).toInt()
            }
            setBackgroundColor(Color.rgb(48, 48, 48))
            loadDataWithBaseURL(
                null,
                ChangeLog.getLog(this@AboutActivity),
                "text/html",
                "UTF-8",
                null
            )
        }
        rootLayout.addView(webView)

        setContentView(scrollView)
    }
}